src/_data/speakers.yml

```diff
@@ -97,40 +116,45 @@
   social_type: twitter
   photo_url: /images/speakers/hitoshi-hasumi.jpg
   bio: Hitoshi, a programmer at Monstarlab, is the creator of PicoRuby & PRK Firmware. He maintains ruby/irb and ruby/reline. Winner of Fukuoka Ruby Award First Prize in 2020 & 2022, and a Ruby Prize 2021 final nominee.
-- id: omar-sotillo-franco
-  first_name: Omar Sotillo
-  last_name: Franco
+
+- id: huy-du
+  first_name: Huy
+  last_name: Du
   type: speaker
-  photo_url: /images/speakers/omar.jpg
-  bio: |
-    Omar, a Staff Software Engineer III hailing from Spain, has embraced the vibrant city of Berlin, for half a decade. Employed at FactorialHR, he ingeniously crafts technical solutions, fearlessly venturing into uncharted territory within the Core: Processes - Communications team
-  social_url: https://twitter.com/geiloco
+  photo_url: /images/speakers/huy-du.jpg
+  bio: Huy is a Software Engineer driven by a genuine passion for his craft. He loves tackling complex challenges in finance and technology. He firmly believes that technology can revolutionize our world in every aspect, and he’s excited about using his skills to make a positive impact.
+  social_url: https://twitter.com/DuGiaHuy
   social_type: twitter
-  website: https://omar.geilo.co
-- id: elle-meredith
-  first_name: Elle
-  last_name: Meredith
-  type: speaker
-  photo_url: /images/speakers/elle-meredith.jpg
-  bio: Elle is a back-end engineer with more than 15 years of experience, leading projects and teams for many technology organisations. At Blackmill, she coaches engineers in effective management practices so that they become engaging, thoughtful and inclusive leaders.
-  social_url: https://tinnies.club/@elle
-  social_type: mastodon
-- id: rishi-jain
-  first_name: Rishi
-  last_name: Jain
+  website: https://dugiahuy.com
+
+- id: katrina-owen
+  first_name: Katrina
+  last_name: Owen
   type: speaker
-  social_url: https://twitter.com/jainrishi15
+  photo_url: /images/speakers/katrina-owen.jpg
+  bio: Katrina has worked as a professional software developer for 18 years. She is the co-author of <a href="https://sandimetz.com/99bottles">99 Bottles of OOP</a>. Katrina is probably most well known for creating <a href="https://exercism.org/">Exercism</a> - a free, not-for-profit coding education platform powered by a wonderful volunteer community, that serves over 1.5 million users.
```

Comment:
We don't normally include hyperlinks on bios

Reply:
It works, though. I made a fix in one if the component: https://github.com/bangkokrb/rubyconfth/pull/335/files#diff-3fe6074553b6b502b27879f0ba145d5cb807e5d6e7c483adcd9ec89188621cbbL23

Reply:
just dont want the other speakers to complain they dont have hyperlinks :D

Reply:
Fair point. Let's got with equality. I have removed the links in 86a40ef ✅